src/components/createBook/CreateBook.tsx

```diff
@@ -1,26 +1,167 @@
+  }
+
   return (
     <div className="single-card">
-      <img className= "cover" src={noCover}/>
-      <div  className='section'>
-        <label className="name-label" >Name</label>
-        <input className = "name-input" type="text"/>
-      </div>
-      <div  className='section'>
-        <label className="desc-label" >Description</label>
-        <input className = "desc-input" type="text"/>
-      </div>
-      <div className='section'>
-        <label className="isbn-label" >ISBN</label>
-        <input className = "isbn-input" type="text"/>
-      </div>
-      <div  className='section'>
-        <label className="quantity-label" >Quantity</label>
-        <input className = "quantity-input" type="number" min={0}/>
+      <div className='all-inputs'>
+        <div className='cover-section'>
+          <div  className='section'>
+            <img className= "cover" src={cover}/>
+            <input type="file" onChange={(event) => handleChange(event)}/>
+          </div>
+        </div>
+        <div className='forms-sections'>
+          <div  className='section'>
+            <label className="name-label" >Title</label>
+            <input className = "form-input" type="text"
+              onChange={(e) => setBook(prevState => ({ ...prevState, Title: e.target.value }))}
+            />
+            <label className='error-message'>{titleErrorMessage}</label>
+          </div>
+          <div className='section'>
+            <label className="isbn-label" >ISBN</label>
+            <input className = "form-input" type="text"
+              onChange={(e) => setBook(prevState => ({ ...prevState, ISBN: e.target.value }))}
+            />
+            <label className='error-message'>{ISBNErrorMessage}</label>
+          </div>
+          <div  className='section'>
+            <label className="desc-label" >Description</label>
+            <textarea className = "form-input" rows={3}
+              onChange={(e) => setBook(prevState => ({ ...prevState, Description: e.target.value }))}
+            />
+          </div>
+          <div  className='section' >
+
+            <label className="authors-label" >Authors</label>
+            <div className='author-section'>
+              <Select
+                isMulti
+                className="authors-select"
+                maxMenuHeight={200}
+                value={selectedAuthors}
+                options={authorList}
+                classNamePrefix="select"
+                getOptionLabel={(option: AuthorResponse) => `${option.FirstName} ${option.LastName}`}
+                getOptionValue={(option: AuthorResponse) => option.Id.toString()}
+                onChange={handleSelectedAuthorsChange}
+              />
+              <button className='add-btn' onClick={() => setShow((s) => !s)}><AddIcon className = "icon" size={40} color="#fce4db" /></button>
+              <CreateAuthorModal setIsAuthorsChanged={setIsAuthorsChanged} show={show} closeModal={() => setShow(false)}/>
```

Comment:
Here you can render modal if show is true, you don't need to pass it down to the modal itself (this is follow up comment for the one in the modal file)